### ISSUE TITLE
Table (Ti ho corretto anche un typo in Location, c'era Loaction)

### ISSUE THREAD

**Issue by mzanella** [annabonaldo/OSNInfoInference#2]
```TeX
\begin{table*}[h]
\label{tab:comparison}
\begin{tabular}{lllllll}
\textbf{Attribute}  & \textbf{work}  & \textbf{year}       & \textbf{accuracy}   & \textbf{method}     & \textbf{OSN} & \textbf{Inference Input}  \\ \hline
\\[-0.2cm] \textbf{Gender} & & & & & & \\[-0.1cm]
& \cite{li2017measurement} & 2017   &  -  & Inference from first name            & Amazon   & User first Names\\  
& \cite{dougnon2015inferring}   & 2015  & 95.70  & Graph Label Propagation (GLP) & Pokec    & SGraph \\ 
 & \cite{dougnon2015inferring}& 2015       & 91.60      & Graph Label Propagation (GLP)               & Facebook & Social Graph\\ \hline
\textbf{Age}  & & & & & & \\ 
 minor/major\footnote{Mean on single minor-major classifications} & \cite{dougnon2015inferring}                          & 2015    & 55.20      & Graph Label Propagation (GLP) & Facebook & SGraph, PubAttr \\ 

  & \cite{dougnon2015inferring}    & 2015    & 2.2\footnote{Mean Error} & Graph Label Propagation (GLP)     & Pokec    & SGraph, PubAttr \\ \hline

\\[-0.2cm] \textbf{Location} & & & & & & \\ 
160km & \cite{cheng2010you} & 2010 & 51.00 & Probabilistic (ML)& Twitter& Text: geo-tags \\
 State level & \cite{eisenstein2010latent} & 2010 & 24.00  & Geographic topic model (NLP)    & Twitter  & NLP\footnote{regional language features},Text: geo-tags\\
Zip code level & \cite{kinsella2011m}  & 2011    & 13.90    & Language Models  & Twitter & Text: geo-tags\\

Town level & \cite{kinsella2011m}& 2011 & 24.80 & Language Models  & Twitter & Text: geo-tags\\
160km & \cite{chang2012phillies}  & 2012  & 49.90 & Gaussian Mixture models and MLE (ML)  & Twitter & Text:geo-tags\\
160km & \cite{li2012towards}& 2012   & 62.30 & Probabilistic (ML) & Twitter & SGraph\\
10km& \cite{ikawa2012location} & 2012  & 20.00& Machine Learning & Twitter  &Text:geo-tags \\
0,1km & \cite{sadilek2012finding} & 2012  & 57.00  & Dynamic Bayesian Networks                   & Twitter  & Imgs, Text:geo-text\\
10km  & \cite{schulz2013multi} & 2013    & 37.00   & Gazzetteer & Twitter  & NLP, geo-text composition \\
10km & \cite{ryoo2014inferring} & 2014  & 60.00& Probabilistic (ML)& Twitter  & NLP \\
 Country level  & \cite{qian2017probabilistic} & 2017 & 95. 81 & Multinomial Log. Regr., Deep L.  & Twitter  & Text:geo-tags\\
 City level  & \cite{qian2017probabilistic}  & 2017       & 59.20      & Multinomial Log. Regr,Deep L.  & Twitter  & Text:geo-tags\\
City level & \cite{qian2017probabilistic} & 2017       & 70.34      & Multinomial Log. Regr,Deep L. & Weibo    & Text:geo-tags\\
City level  & \cite{davis2011inferring} & 2011       & 88.60      & Graph Label Propagation (GLP)               & Facebook &\\
City level & \cite{qian2017probabilistic}& 2017   & 91.52   & Multinomial Log. Regr,Deep L. & Facebook & Text: locality tags\\ \hline
\\[-0.3cm] \textbf{Language} \footnote{Mean for Italian, Spanish, French, German classifications}   & & & & & & \\       
& \cite{dougnon2015inferring}                          & 2015       & 80.42      & Graph Label Propagation (GLP)               & Pokec   & SGraph, PubAttr \\ \hline
\\[-0.3cm] \textbf{Habits,Education}: & & & & & & \\ 
Driker & \cite{dougnon2015inferring}   & 2015   & 65.42   & Graph Label Propagation (GLP)     & Pokec    & SGraph, PubAttr \\ Smoker& \cite{dougnon2015inferring}  & 2015       & 71.65      & Graph Label Propagation (GLP)               & Pokec    & SGraph, PubAttr \\ 
Music  \footnote{Mean for Metal, Rock, HipHop, Rap classfication}          & \cite{dougnon2015inferring}  & 2015       & 73.09      & Graph Label Propagation (GLP)               & Pokec    & SGraph, PubAttr,  \\
Education & \cite{cascavilla2015revealing, cascavilla2016ossint} & 2016, 2017 & 75.00      & Graph Label Propagation (GLP)              &  &     SGraph, PubAttr       \\
Education  & \cite{dougnon2015inferring}                          & 2015       & 16.70      & Graph Label Propagation (GLP)               & Facebook   & SGraph, PubAttr 
\end{tabular}
\end{table*}


\section{Sensitive attribute disclosure} \label{attribute}
User attributes are usually considered in information security and privacy laws as Personally identifiable information (PII). \textit{Quasi-identifiers} are instead private attributes that are not of themselves unique identifiers, but are sufficiently well correlated with an entity that they can be combined with other quasi-identifiers to create a unique identifier. It has been shown they can be used on its own or with other information to identify, contact, or locate a single person, or to identify an individual in context\cite{pii}. Many inference technique to infer such kind of information has been presented in works in \ref{tab:comparison}. High accuracy techniques are aviable for basic information, expecially gender and age. But also location, education language and habits can be decided with good accuracy results from highly aviable data. Recent works show that also more complex and sensitive information can be inferred from OSN profiles. Most relevant results have been summarized in following lines. 

\subsection{Credit profile inference} \label{creditprofile}  
User credit is a particularly private attribute, usually more sensitive than age or gender in most cases. Users seldom generate credit related personal data on the social web. Consequently, social data only contains extremely weak signals about user credit risk \cite{guo2016personal}. 
Work in \cite{guo2016personal}, based on Twitter platform, perform an analysis on behavioural data that are usually more precise and formal than tweets content itself, and reflects users’ behaviour habits and characters more comprehensively and directly. 

Credit profiling task has been performed in \cite{guo2016personal} using standard l2-regularized logistic regression classifier whose input is the latent user behaviour dimensions. 
After the classifier learning phase, they could distinguish behaviour dimensions that are informative for credit prediction from the classifier easily. Their experiments achieve an accuracy around 63.5\% in predicting credit profiles on available data. 

\subsection{Overweight status inference}
  Work in \cite{kocabey2017face}\textbf{\textbf{}} uses a computer-vision approach to make an evaluation of user body mass index using profile images. 
  Profiles' pictures are available in most cases for OSN profiles, even for more popular social platforms. 
  Inferring basic attributes from them is considered as a privacy attack, but we can assume that 
  getting health information from a so far available resource should be considered a more serious privacy issue. 
  The considered work starts form a dataset of 4206 faces photos collected on \textbf{Reddit}. The experiments aims to distinguish between some BMI ranges that are: [16.0 - 18.5], [18.5, 25],[25, 30], [30, 35], [35, 40], [$\ge$ 40]. 
  Male-famale subdivision in photo dataset was of 2438 males and 1768 females.
  For training phase 3368 images where used, while 838 were implied in testing phase.    
  The experiment results shows a precision of 0.71 for male face BMI classification and of 0.57 for female BMI classification, considering the most performing version of purposed algorithms.  
  As last phases of the study, the same classification task over here has been submitted to humans. Then accuracy results have be compared. The accuracy difference between human and machine performances were really similar. Humans outperforms machines in most cases, but with a greater accuracy of at most 5\%.
 \subsection{Inference on mental-health conditions}\label{mentalhealth}
  In \cite{benton2017multitask} authors develop neural MTL models for prediction tasks on users' mental health status.  They work on 10 classification task: 8 on mental health conditions, one on neurotypicality presence and gender classification. 
  Considered mental health condition are: neurotypicality, anxiety, depression, suicide attempt, eating disorder, schizophrenia, panic disorder, bipolar disorder, PTSD. 
  The total training set size was about 9611 users, divided as 788 males and 248 females. 
  They opt for a neural architecture to exploit the synergies
  between mental conditions. They chose a model that allow to get
  improvements over single task models by
  using MTL (Multiple Task Learning) that is predicting multiple related tasks and that allow to exploit any correlations between the predictions.
  MTL model performance was compared with STL (Single Task Learning) models to predict each task independently.
  Results show that the most complex MTL model performs significantly better than independent LR models, reaching 0.846 \textit{True Positive Rate} where \textit{False Positive Rate}=0.1.
  Choosing the correct set of auxiliary tasks for a
  given mental condition can yield a significantly
  more predictive model. 
  The MTL model used in this work seems to be really helpful in improving performance for training with small amount of data. We can see that in STL case the performances are badly affected for small training dataset. Otherwise MLT give better performances even for categories with really low number of training examples. 
  AUC (Area Under Curve) measure shows an good performance on tasks, but it is not really high (they get at most a \textit{True Positive Rate}=0.81). Otherwise we should consider that this is not a simple classification task and, with that point ov view, we should consider this a really hopeful inference result. 
  What not seems so clear is kind of input data experiment uses. They say they uses all uses history on Twitter, but no implementation detail is given. As reported, the relative frequency of the 5,000 most frequent
  n-gram features, for $ n~\in~ {1, 2, 3, 4, 5}  $in  data,has been computed 
  and then feed as input to all models. 
  Otherwise, this procedure seems to be a core feature for successful implementation, and is difficult to analyse in presence of no other information. 
  
  \subsection{Personality traits inference}\label{personality}
  
  A work in\cite{segalin2017your} tries to understand if profile pictures contain information about their owners’ personality. They found the amount of significant correlations found was considerable, which supports the previous other works findings that personality traits have an influence on the choice of profile pictures in Facebook. 
  Multiple feature extraction has been executed for tree main features categories:colour, composition and textural properties. 
  They apply different systems to perform features extraction such as Computational Aesthetics (CA), Convolutional Neural Networks(CNN) and other two feature extraction systems that are IATO \cite{celli2015iato} and Pyramid Histogram Of visual Words based features. Best accuracy in features extraction task were obtained combining all experimented methods together and is of \textbf{60\% for all 5 personality profile considered.} 
  A later work on personality feature extraction from images of social networks profiles that uses a similar 5-grade personality classification model get better results in classification of profiles. 
  Achieved accuracy improves previous results of a percentage that starts from +0.8\%  up to +15\%. 
  Result in \cite{celli2015iato,segalin2017your} where again outperformed by \cite{celli2017profilio} that develops a complex framework for personality classification using social media data.  They has developed a profiling technology that perform prediction for customers profiling for advertising customization scopes. Their product is based on different sources of digital footprints (e.g. images, textual contents, demographics,  social media activities, etc.), and is able to predict a set of behavioural variables, including purchase motivations, job performance and subjective well-being. They underline the importance of exploiting public information and using mixed typologies of data sources. They refer their system reach an accuracy of 80\% in predicting personality traits.
  Even other recent works \cite{gelli2017personality} exploit personality prediction from social media data for services customization, enforcing that effectively predicting personality traits is today a feasible task. In particular work in \cite{buraya2017towards} reports accuracy value achieved for personality traits prediction, comparing accuracy for predictions on single OSN data and multiple OSN  data. From their results we can see that using multiple source of data increase in a sensible way accuracy of predictions. Otherwise they point out that it is not always true that the more data we get, the more is the accuracy. Their experiments results shows that in each experiment set-up, pairs of OSN data association always outperform tree-OSN data association.

**Comment by annabonaldo** [comment]
graaazie
